docs/tendermint-core/state-sync.md

```diff
@@ -9,3 +9,9 @@ With state sync your node will download data related to the head or near the hea
 This leads to drastically shorter times for joining a network. 
 
 Information on how to configure state sync is located in the [nodes section](../nodes/state-sync.md)
+
+## The State Sync event
+When Tendermint Core launches and the statesync flag has been enabled in the config file, it might start the state-sync to fetch the chain state data from the network. During the starting / finished the data fetching progress, it will emit an event to expose the state-sync `complete` status and the state `height`.  
```

Comment:
```suggestion
## The State Sync event

When Tendermint Core launches and the statesync flag has been enabled in the config file, it might start the state-sync to fetch the chain state data from the network. During the starting / finished the data fetching progress, it will emit an event to expose the state-sync `complete` status and the state `height`.  
```